app/labor/email_logic.rb

```diff
@@ -28,13 +27,19 @@ def should_receive_email?
 
   def get_articles_to_send
     fresh_date = get_fresh_date
+
     articles = if user_has_followings?
+                 experience_level_rating = (@user.experience_level || 5)
+                 experience_level_rating_min = experience_level_rating - 3.6
+                 experience_level_rating_max = experience_level_rating + 3.6
```

Comment:
to avoid a super long line and also on the tune of "explicit is better than implicit"